src/spec-node/devContainersSpecCLI.ts

```diff
@@ -122,6 +122,7 @@ function provisionOptions(y: Argv) {
 		'dotfiles-install-command': { type: 'string', description: 'The command to run after cloning the dotfiles repository. Defaults to run the first file of `install.sh`, `install`, `bootstrap.sh`, `bootstrap`, `setup.sh` and `setup` found in the dotfiles repository`s root folder.' },
 		'dotfiles-target-path': { type: 'string', default: '~/dotfiles', description: 'The path to clone the dotfiles repository to. Defaults to `~/dotfiles`.' },
 		'container-session-data-folder': { type: 'string', description: 'Folder to cache CLI data, for example userEnvProbe results' },
+		'skip-persisting-remote-env-from-config': { type: 'boolean', default: false, description: 'Do not persist remoteEnv from devcontainer.json in container metadata label' },
```

Comment:
Add `hidden: true`, so we don't propagate the use of this workaround.

`persisting` is maybe to general (we still write the env cache in the container - is that an issue?). `omit-config-remote-env-from-metadata` may be clearer?

Reply:
I prefer 'omit' as well

Reply:
Sounds good. Updated